objectsapiclient/models.py

```diff
@@ -45,7 +45,10 @@ def __str__(self):
 
     @cached_property
     def client(self):
```

Comment:
Given that this will be evaluated at module load, we'll want to ensure the caching here doesn't go stale. Let's just remove the caching decorator until performance becomes an issue.

```suggestion
    def client(self):
```

Reply:
Sorry, should've said `@cached_property -> @property`